setup.py

```diff
@@ -230,6 +230,10 @@ def prepare_env_for_osx():
         description=
         """Python API for cpptraj: a data analysis package for biomolecular simulation""",
         license="GPL v3",
+        install_requires=['numpy', 'libmagic'],
+        tests_require=['tqdm', 'traitlets', 'IPython', 'mpi4py', 'pytest', 'cclib',
+                       'pysander', 'ipywidgets', 'nglview', 'ipykernel'],
+        setup_requires=['Cython'],
```

Comment:
will `Cython` be pulled if user does `pip install pytraj`. We have binary wheel for it.

Reply:
Yes and no. If the user installs `pytraj` from a binary package then Cython is not installed at all. If the user installs from source (either via git clone and then `python setup.py build` or via a source package as `pip install --no-binary pytraj`) then Cython is installed in the temporary build environment during the build process, but does not end up in the current Python environment.

Reply:
there is a complication here. If `pytraj` is distributed via `ambertools,` the cythonized files are included so user does not need cython at all.

Reply:
Ok. There's no harm in leaving Cython in `setup_requires` in that case. It may be installed in vain, but it's removed again without the user noticing. No packages in `setup_requires` end up in the final environment.

Reply:
I think I would skip the line `setup_requires=['Cython'],` for now. It's too complicated within the ambertools context. (ambertools is a program suite and Python is not its main language).

Reply:
Done.